lib/Network.chpl

```diff
@@ -774,17 +800,19 @@ class Sequential : Module(?) {
         init this;
         if overrideName then
             this.moduleName = moduleName;
+        else
+            this.moduleName = "sequential";
```

Comment:
given you added defaults for the other `inits` to have a `moduleName` of `sequential`, should this `init` also have that instead of the empty string?

Reply:
I'm not quite sure why this overload has a `param overrideName` so I didn't mess with it. But yes, ideally we can just have the default value of `moduleName` as `sequential`. I will make the change